### DISCUSSION TITLE
Wireless control

### DISCUSSION THREAD

**Discussion by MrPeens** [gpstar81/GPStar-proton-pack#236] [Q&A] [answered]
I'm finally starting my pack mod. I have no interest in the attenuator but love the wireless control option. Is there any chance of a new version of the pack board with wireless built in? If no, is there room in the Haslab pack for the attenuator board to be added in? 
Thanks so much for all the incredible work on this project!

**Comment by gpstar81** [comment, marked as answer]
No plans for a new board with WiFi built in, as the original was designed with the ability to expand by connecting additional boards and devices to it. 

You can easily fit the Attenuator board into the pack no problem, there is lots of room for one. Or a bare bones wireless adapter board as indicated here:

https://github.com/gpstar81/GPStar-proton-pack/blob/main/WIRELESS.md
